include/kernel/syscall.h

```diff
@@ -43,6 +43,7 @@ typedef enum {
 	SYSCALL_RMDIR,
 	SYSCALL_READDIR,
 	SYSCALL_PWD,
+	SYSCALL_DUP_VOLUME,
```

Comment:
Let's use "copy" instead of "dup" throughout, since "dup" already refers to a shallow copy of a kobject descriptor.